docs/common-questions.md

```diff
@@ -1,9 +1,17 @@
 # Common Questions
+## API Documentation (JavaDoc)
```

Comment:
You probably wanted to remove the _Appendices_ area which only hosted this section but I'm not sure if _Common questions_ is a good place for it. Since we encourage people here only to use the JanusGraph API for management purposes, why not move it into the new _Operations_ area?